Plugins/Lint Source Code/plugin.swift

```diff
@@ -0,0 +1,92 @@
+import PackagePlugin
```

Comment:
Same comments from the Format Source Code file apply here (factoring out the common implementations, making a single invocation to swift-format, accessing the Xcode project directory).